include/exadg/postprocessor/kinetic_energy_spectrum.cpp

```diff
@@ -593,20 +593,18 @@ KineticEnergySpectrumCalculator<dim, Number>::setup(
   dealii::DoFHandler<dim> const &         dof_handler_in,
   KineticEnergySpectrumData const &       data_in)
 {
-  data        = data_in;
-  dof_handler = &dof_handler_in;
-
-  AssertThrow(
-    dof_handler->get_triangulation().all_reference_cells_are_hyper_cube(),
-    dealii::ExcMessage(
-      "This postprocessing utility is only available for meshes consisting of hypercube elements."));
-
-  clear_files = data.clear_file;
-
-  time_control.setup(data_in.time_control_data);
-
   if(data_in.time_control_data.is_active)
   {
+    data        = data_in;
+    clear_files = data.clear_file;
+    time_control.setup(data_in.time_control_data);
```

Comment:
one line above, `data` is used, so I would also use `data` here instead of `data_in`.